vmdb/app/models/container_node.rb

```diff
@@ -5,16 +5,16 @@ class ContainerNode < ActiveRecord::Base
   # :name, :uid, :creation_timestamp, :resource_version
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many   :container_groups
-  has_many   :container_node_conditions, :dependent => :destroy
+  has_many   :container_conditions, :class_name => ContainerCondition, :as => :container_entity, :dependent => :destroy
   has_one    :computer_system, :as => :managed_entity, :dependent => :destroy
   belongs_to :lives_on, :polymorphic => true
 
   delegate   :hardware, :to => :computer_system
 
-  virtual_column :ready_condition_status, :type => :string, :uses => :container_node_conditions
+  virtual_column :ready_condition_status, :type => :string, :uses => :container_conditions
```

Comment:
@zeari yes, I have seen the use of virtual columns in several areas of the project (EmsCluster list view for example).
Currently there's only one condition, but notice I only retrieve the one with the name 'Ready' so we are covered in the case of adding more conditions in the future.